planning/obstacle_velocity_limiter/launch/obstacle_velocity_limiter.launch.xml

```diff
@@ -0,0 +1,29 @@
+<launch>
+  <!-- input/output -->
+  <arg name="input_trajectory" default="/planning/scenario_planning/scenario_selector/trajectory"/>
+  <arg name="input_dynamic_obstacles" default="/perception/object_recognition/objects"/>
+  <arg name="input_occupancy_grid" default="/perception/occupancy_grid_map/map"/>
+  <arg name="input_obstacle_pointcloud" default="/perception/obstacle_segmentation/pointcloud"/>
+  <arg name="input_odometry" default="/localization/kinematic_state"/>
+  <arg name="input_map" default="/map/vector_map"/>
+  <arg name="output_trajectory" default="~/output/trajectory"/>
+  <arg name="output_debug_markers" default="~/output/debug_markers"/>
+  <arg name="use_sim_time" default="false"/>
```

Comment:
No need since it is defined by autoware.launch

Reply:
Done: 2bcb16ccf